jest.config.ts

```diff
@@ -15,6 +15,6 @@ export default async (): Promise<Config> => {
     moduleNameMapper: {
       '\\.(css|less)$': 'identity-obj-proxy'
     },
-    transformIgnorePatterns: ['node_modules/(?!(multiformats|uint8arrays|@dcl/single-sign-on-client|decentraland-connect)/)']
+    transformIgnorePatterns: ['node_modules/(?!(multiformats|uint8arrays|@dcl/single-sign-on-client|decentraland-connect|uuid)/)']
```

Comment:
I think you can ignore this by installing a previous version of uuid, similar to what happens with node-fetch.

Just informational :D

Reply:
UUID is a dependency of a dependency, meaning that we can't downgrade it :(